cmds.go

```diff
@@ -216,12 +224,46 @@ func printExport(socket, certPath string) {
 	}
 }
 
+func printExportWindows(values map[string]string) {
+	// Print cmd.exe instructions to stderr
+	fmt.Fprintln(os.Stderr, "If you are running inside Windows Command Prompt (cmd.exe), copy and paste the")
+	fmt.Fprintln(os.Stderr, "following commands to your terminal to set the environment variables:")
+	for name, value := range values {
+		if value == "" {
+			fmt.Fprintf(os.Stderr, "    set %s=\n", name)
+		} else {
+			fmt.Fprintf(os.Stderr, "    set %s=%s\n", name, value)
+		}
+	}
+	fmt.Fprintln(os.Stderr, "")
+	// Print powershell instructions to stderr
+	fmt.Fprintln(os.Stderr, "If you are running inside PowerShell, copy or paste the following commands")
+	fmt.Fprintln(os.Stderr, "to your shell or run \"boot2docker shellinit | Invoke-Expression\" to set the")
+	fmt.Fprintln(os.Stderr, "environment variables:")
+
+	// Print powershell exports to stdout
+	for name, value := range values {
+		if value == "" {
+			fmt.Printf("    Remove-Item Env:\\%s\n", name)
+		} else {
+			fmt.Printf("    $Env:%s = \"%s\"\n", name, value)
```

Comment:
@gmlewis this particular one is easy to fix. I'll just fix. Problem was mostly above where I can't have `\n` inside backticks with fmt.Fprintf. thanks.